catalog/docs/src/main/resources/ManContents.adoc

```diff
@@ -182,3 +182,39 @@ No configuration is required.
 (IMG)
 
 . Click delete.
+
+
+== Metacard Validation Plugin
+
+The metacard validation plugin (`catalog-plugin-metacard-validation`) is a feature that allows the validation/invalidation of metacards when they are ingested.
+Without this plugin, if validation is enabled, metacards are either valid and allowed into the catalog or are invalid and disallowed into the catalog. Without this plugin, if validation is disabled,
+all metacards will be valid and allowed into the catalog. With this plugin, validation can be enabled, and then metacards will be marked by this plugin as valid or invalid.
+
+=== Enabling feature
+
+This feature is turned off by default and can be enabled by typing `features:install catalog-plugin-metacard-validation` in the command line while DDF is running.
+
+=== Configuring feature
+
+The feature has one configuration, `Enforced validators`. `Enforced validators` are a set of metacard validators chosen by an administrator or knowledge manager to disallow invalid metacards from being stored in the catalog.
+The value of the field should be the `id` of the validator or the fully qualified class name.
+
+For example, an administrator could want to allow _most_ invalid metacards but prevent any metacards that fail security validation from being ingested. They would input the security validator's id into the `enforced validators` field, and then
+any metacard that failed that validator would *NOT* be stored in the catalog.
```

Comment:
Are these validators services that need to be installed? If so, we should have an SDK example of a validator so that people can make their own.